hardware/EvohomeWeb.cpp

```diff
@@ -77,7 +77,7 @@ void CEvohomeWeb::Init()
 
 bool CEvohomeWeb::StartSession()
 {
-	_log.Log(LOG_NORM, "EvohomeWeb: connect to Evohome server");
+	_log.Log(LOG_NORM, "EvohomeWeb: start new session with Evohome server");
```

Comment:
This will result in 24 log messages a day ? Maybe an option to silence this log for a session update ?